components/BuildPage/Hero.jsx

```diff
@@ -50,7 +50,7 @@ const Hero = () => (
 
         <div className="flex flex-wrap justify-stretch gap-6">
           <Button variant="default" size="xl" asChild className="grow">
-            <a href={CTA_LINK}>Get started</a>
+            <Link href="#grow">Get started</Link>
```

Comment:
unless there's some special styling for this, there's no need for `Link`

`a` is better as it's not routing to another page, and is much more lightweight than `Link`

(reason why we use `Link` for internal links (to other pages) is because there's a bunch of extra routing magic that goes on in the background)

```suggestion
            <a href="#grow">Get started</a>
```